lib/domain/rent_charge.rb

```diff
@@ -0,0 +1,25 @@
+module Domain
+  class RentCharge
+    attr_accessor :uprn,
+                  :address,
+                  :address_2,
+                  :comments,
+                  :rr_count,
+                  :property_type,
+                  :base_data_bed_size,
+                  :bedroom_weight,
+                  :mra_archetype,
+                  :jan_1999_asset_values,
+                  :year,
+                  :removed,
+                  :formula_rent_this_year,
+                  :rent_cap_this_year,
+                  :uprated_actual
+    
+    def initialize(**rent_charges)
```

Comment:
`rent_charges` here is a hash of data for a single rent charge, while the variable name suggests an array of rent charge objects to me. Is there a better name we could use?

Reply:
good point, the only thing I can think of right now is `rent_charge`, but feel like I'm using this a lot. Unless it's convention to use `config` like you have in your example above?